src/System.Data.SqlClient/tests/Tools/TDS/TDS/Done/TDSDoneTokenCommandType.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.SqlServer.TDS.Done
+{
+    /// <summary>
+    /// Command completion of which a DONE token reports
+    /// See .../sql/ntdbms/ntinc/tokens.h for a complete list of token identifiers
```

Comment:
Will remove. Found more instances of header file references which I will remove.